server/blob/blob-union/src/test/java/org/apache/james/blob/union/UnionBlobStoreTest.java

```diff
@@ -91,6 +91,11 @@ public InputStream read(BucketName bucketName, BlobId blobId) {
             return Mono.error(new RuntimeException("broken everywhere"));
         }
 
+        @Override
+        public Mono<Void> delete(BucketName bucketName, BlobId blobId) {
+            return Mono.error(new RuntimeException("broken everywhere"));
```

Comment:
What does this message mean ?

Reply:
Reviewed the NO_REVIEW commit as well, from https://github.com/linagora/james-project/pull/2524... 

Well means it's broken, all other methods return the same, for test classes that are simulating failing BlobStores. I just kept the consistency here with the rest of those test classes. You can check the all `UnionBlobStoreTest` class if you want to see the picture :)